### PR TITLE
Version 1

### PR DESCRIPTION
**Pull request by mnekx**
## Description

In this version, the following was done:

## Type of change

- [x] Set up the project with Webpack and Jest
- [x] Add tests for items counter
- [x] Find external API
- [x] Add footer
- [x] Add header
- [x] Choose assets
- [x] Display list of items on the Homepage
- [x] Add all items counter on the Homepage
- [x] Display Comments pop up with selected item's details
- [x] Display number of likes for each item on the Homepage
- [x] Display comments for a given item on the Comments pop-up
- [x] Create feature: add  new like
- [x] Create feature: add  new comment
- [x] Add comments counter
- [x] Add tests for comments counter
- [x] Launched to Github pages [here](https://harlexkhal.github.io/Foodipy/)
- [x] Link to Video documentation [here](https://drive.google.com/file/d/1sQjpusrQpeI1nw8gZ0N53OXs87E4Kxy9/view?usp=sharing)

## Checklist:

- [x] No linter errors.
- [x] I have used correct Gitflow.
- [x] Work is documented in a professional way.
- [x] List of JavaScript best practices is followed.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  </head>
+  <body>
+    <header>
+      <nav class="navbar container d-flex">
+        <div class="logo">
+          <a href="index.html"><h1>Foodipy</h1></a>
+        </div>
+        <ul class="nav-menu d-flex">
+          <li class="nav-item"><a href="index.html">View more</a></li>
+          <li class="nav-item"><a href="index.html"><span class="item-count">0</span> Meal(s) found!</a></li>
```
Comment (ShoiraTa) [inline src/index.html:21]: - [ ] Based on project requirements, the page should have a navigation similar to the given wireframe. Pay attention that the wireframe has 3 pages, Spaces, Planets, Races:
Screenshot 2022-03-10 at 18 33 49

Please add 3 pages based on some category, for example, you can make deserts, vegetarian, seafood, like I did in my capstone project, feel free to check it [here](https://shoirata.github.io/RecipeApp/)

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi team,

:thumbsup: This your project is looks great! All the functionality work.

There is only 1 issue that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (harlexkhal) [reply src/index.html:21]: hi @ShoiraTa, I thought the wireframe was there as a guide, not a requirement. the view more on the navigation navigates between more than just 3 categories, when you click on view more or any of the navigations it randomly selects a new category. Thanks for the review

Comment (harlexkhal) [reply src/index.html:21]: Thanks @ShoiraTa

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @mnekx, @harlexkhal 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

**Pull request merged**